libraries/AP_LocationDB/AP_LocationDB.h

```diff
@@ -0,0 +1,139 @@
+#pragma once
+
+#include "AP_LocationDB_config.h"
+
+#if AP_LOCATIONDB_ENABLED
+
+#include <AP_Common/AP_Common.h>
+#include <AP_Common/Location.h>
+#include <AP_Math/AP_Math.h>
+#include <AP_HAL/AP_HAL_Boards.h>
+#include <AP_HAL/Semaphores.h>
+#include <AP_Vehicle/AP_Vehicle_Type.h>
+#include <AC_Avoidance/AP_OABendyRuler.h>
+
+class AP_LocationDB_Item {
+public:
+    friend class AP_Filesystem_LocationDB;
+    friend class AP_LocationDB;
+
+    AP_LocationDB_Item() {};
+    AP_LocationDB_Item(uint32_t key, uint32_t timestamp_ms, Vector3f pos_cm_NEU, Vector3f vel_cm_NEU, Vector3f acc_cm_NEU, float heading_cdeg, float radius_cm, uint8_t populated_fields);
```

Comment:
All arguments passed by value can have added const if you don't plan to change them within the scope of the constructor. 
```suggestion
    AP_LocationDB_Item(const uint32_t key, const uint32_t timestamp_ms, const Vector3f pos_cm_NEU, const Vector3f vel_cm_NEU, const Vector3f acc_cm_NEU, const float heading_cdeg, const float radius_cm, const uint8_t populated_fields);
```

Reply:
Done. Thanks.